sdk/eventhub/azure-eventhubs/azure/eventprocessorhost/partition_pump.py

```diff
@@ -149,7 +149,7 @@ def is_closing(self):
             except Exception as err:  # pylint: disable=broad-except
                 await self.process_error_async(err)
                 if self.host.eph_options.release_partition_on_checkpoint_failure and str(err) == "failed to persist checkpoint":
-                    log.error("Unable to persist checkpoint. This often means we lost the lease. Proactively closing to avoid churn.")
+                    _logger.error("Unable to persist checkpoint. This often means we lost the lease. Proactively closing to avoid churn.")
```

Comment:
>_< thank you....

Reply:
I haven't run a git blame, but I'm not convinced it wasn't me :P  (The beauty of it was that it would cause a failure to bubble up resulting in the right behavior, if through the wrong vector, so we effectively dodged that one)